services/types.ts

```diff
@@ -0,0 +1,30 @@
+import { Socket } from 'socket.io';
+
+export interface Classrooms {
+  [classroomName: SocketId]: {
+    teacherSocketId: string;
+    students: SocketId[];
+  };
+}
+
+export interface Teachers {
+  [teacherSocketId: SocketId]: {
+    socket: Socket;
+    classroomName: string;
+  };
+}
+
+export interface Students {
+  [studentSocketId: SocketId]: {
+    socket: Socket;
+    classroomName: string;
+    realName: string;
+    peerSocketId: SocketId | null;
+  };
+}
```

Comment:
Let's also export Student as its own interface. That way it can be used as a type in the functions `deleteChat()` and `remStudentFromClassroom()`

```suggestion
export interface Students {
  [studentSocketId: SocketId]: Student;
}

export interface Student {
  socket: Socket;
  classroomName: string;
  realName: string;
  peerSocketId: SocketId | null;
}
```